### PR TITLE
[#47362] Replace spot-icon with boxed icons

### PR DESCRIPTION
**Pull request by Kharonus**
- https://community.openproject.org/work_packages/47362
- added new icon font for spot icons
- replaced container containing spot-icon with `inline-flex`

### CODE REVIEW

**Review by marcalcobe — CHANGES_REQUESTED** [review]
Discussed a couple of alignments and fixes in a feedback session with @Kharonus. Includes:
- Modal dialogs header icons
- Invite users in the "+" dropdown

**Comment by Kharonus** [comment]
Fixed!

File: frontend/src/app/core/main-menu/main-menu-navigation.service.ts [hunk 1]
```diff
@@ -86,7 +86,7 @@ export class MainMenuNavigationService {
     // 1. unbind the current click functions
       .unbind('click')
       // 2. wrap each in a span that we'll use for the new click element
-      .wrapInner('<span class="ellipsis"/>')
+      .wrapInner('<span class="ellipsis" style="display: flex; align-items: center"/>')
```
Comment (b12f) [inline frontend/src/app/core/main-menu/main-menu-navigation.service.ts:89]: I'm not sure about this, it'd be nicer to fix it with a class

Comment (Kharonus) [reply frontend/src/app/core/main-menu/main-menu-navigation.service.ts:89]: definitely, but ellipsis is a global class. Should we introduce a global class that enables an inline-flex container?

**Pull request merged**